Felienne/hedy
# [FEATURE] Adds tags to public profiles of users

**Description**
In this PR we add a new and exiting feature: profile tags! From now on users will be able to receive tags that will be shown on their public profile page. The "teacher" and "admin" tag are automatically added on updating the public profile, other tags can be added by the admin. At this moment we have the following tags:

- Admin
- Teacher
- Certified teacher
- Distinguished user
- Contributor

**Fixes**
This PR fixes #3149.

**How to test**
Make sure you are an admin and navigate the the `/admin/users` page. Notice the new `tags` column on the table. Verify that clicking the icon displays a modal with all the possible tags enabling the admin to remove / add tags. Also verify that adding and removing tags is correctly stored on the public profile of the user. When no profile exists an error is returned.
